pkg/worker/worker.go

```diff
@@ -213,6 +213,14 @@ func (w *Worker) NewService(name string) *Service {
 	return svc
 }
 
+func (w *Worker) RegisterWorkflow(workflow workflowConverter) error {
+	wf, ok := workflow.(*WorkflowJob)
```

Comment:
seems like of the possible values of `workflowConverter` only `WorkflowJob` can have triggers